cleanlab/datalab/internal/issue_manager_factory.py

```diff
@@ -201,7 +201,14 @@ def list_default_issue_types(task: str) -> List[str]:
     :py:class:`REGISTRY <cleanlab.datalab.internal.issue_manager_factory.REGISTRY>` : All available issue types and their corresponding issue managers can be found here.
     """
     if task == "regression":
-        default_issue_types = ["label"]
+        default_issue_types = ["label", "null"]
```

Comment:
Just focus on classification in this PR.

```suggestion
        default_issue_types = ["label"]
```